### DISCUSSION TITLE
Can we integrate hasura with EDB Biganimal Database.

### DISCUSSION THREAD

**Discussion by TheDataGuyEDB** [hasura/graphql-engine#9176] [Question]
Can we integrate hasura with EDB Biganimal Database?

**Comment by adas98012** [comment]
Thanks for your query.  This is still work in progress and we'll have updates from our product team on this. 
Meanwhile, we would like to share our current [test matrix ](https://github.com/dventimihasura/hasura-projects/blob/master/edb-test-1/README.org#5-test-matrix) with various EDB Databases.

**Comment by ajohnson1200** [comment]
We have official documentation for this EDB BigAnimal here:

https://hasura.io/docs/latest/databases/connect-db/cloud-databases/enterprisedb/
